src/app/containers/RadioSchedule/utilities/testHelpers.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import RadioScheduleContainer from '..';
+import { RequestContextProvider } from '#contexts/RequestContext';
+import { ServiceContextProvider } from '#contexts/ServiceContext';
+import { ToggleContext } from '#contexts/ToggleContext';
+
+const localRadioScheduleEndpoint = service => {
```

Comment:
I don't think we need this function. Looks like we are only using it for our RadioSchedulesWithContext component below but that is only used in unit tests which will have mock fetched data.

Reply:
Removed `localRadioScheduleEndpoint()`

Reply:
Thanks Ruth 💯